samples/ChatSample.Cli/Program.cs

```diff
@@ -0,0 +1,592 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
add a Readme.md explaining the usage of the cli, and show a typical use scenario of it?